extension/extension/background.js

```diff
@@ -96,6 +98,16 @@ function websocket_logger(server) {
             event = JSON.stringify(event);
             queue.push(event);
         };
+        socket.onmessage = function(event) {
+            const message = event.data;
+            console.log('Received message:', message);
+            // Extract the auth message from the server
+            var authStatusMsgPattern = /STATUS:\s*(\w+)/;
+            const match = message.match(authStatusMsgPattern);
+            if (match) {
+                authStatus = match[1];
+            }
```

Comment:
We should continue to use JSON to communicate. 

For future reference, regexp should be avoided in production code. It is fragile and also tends to leave things open to attacks. This use is okay from a security perspective, but the upside of using something standard is it doesn't need to be verified (and it makes a big difference as code is changed and mutated).